vic/drivers/shared_all/src/vic_time.c

```diff
@@ -851,7 +853,21 @@ time_delta(dmy_struct        *dmy_current,
             }
         }
         else if (freq == FREQ_NYEARS) {
+            // advance year
             dmy_next.year += n;
+            // re-calculate day_in_year (NOTE: currently will have problem
+            // if dmy_next.year is not leap year but date is Feb 29 !)
```

Comment:
Clarify comment. It's unclear what `currently` refers to. Is that before or after the fix. If before, then it does not belong here (since you can put this info in the commit message). If after then it seems we still have a problem.